packages/teams-js/src/internal/messageObjects.ts

```diff
@@ -16,7 +64,8 @@ export interface MessageRequest {
  * Limited to Microsoft-internal use
  */
 export interface MessageResponse {
```

Comment:
>MessageResponse [](http://example.com/codeflow?start=17&length=15)

I would have expected to see a matching `SerializedMessageResponse` object as well -- would it be possible to have one for all the same type safety reasons as we have above?

Reply:
Since the data in the `MessageResponse` object is being used immediately, I elected to not make a `SerializedMessageResponse` and `MessageResponse` object since the UUID for example is being sent as a UUID object across our code rather than as two separate `MessageResponse` objects.

For extensibility though, I think I will go back and change this into a serialized and deserialized object.